chart/values.yaml

```
@@ -701,6 +701,10 @@ webserver:
     #     port: 80
     #     targetPort: 8888
     loadBalancerIP: ~
+    ## Limit load balancer source ips to list of CIDRs
+    # loadBalancerSourceRanges:
+    #   - "143.231.0.0/16"
```

```suggestion
    #   - "10.123.0.0/16"
```

Thoughts on the examples being a private block instead?

---

I put it only at example, fixing this in next commit